modules/howtos/pages/n1ql-queries-with-sdk.adoc

```diff
@@ -2,10 +2,8 @@
 :navtitle: N1QL from the SDK
 :page-topic-type: howto
 :page-aliases: n1ql-query,ROOT:querying-n1ql
-:lang: Java
-:version: 3.1.3
 :example-source: 3.1@java-sdk:howtos:example$Queries.java
-:example-source-lang: java
+include::howtos:partial$attributes.adoc[]
```

Comment:
Please link to https://github.com/couchbase/docs-sdk-java/blob/release/3.1/modules/project-docs/partials/attributes.adoc
include::project-docs:partial$attributes.adoc[]
we removed all of the others so that there is only one to update with each patch release ;)

Reply:
No problem, will fix!